core/src/main/resources/application.properties

```diff
@@ -18,12 +18,12 @@
 
 #Uncomment the below to configure Spring JPA properties to connect Klaw to Postgres Rdbms
 
-#spring.datasource.url=jdbc:postgresql://localhost:5432/klaw?cachePrepStmts=true&useServerPrepStmts=true&rewriteBatchedStatements=true
+#spring.datasource.url=jdbc:postgresql://localhost:5432/klawdb?cachePrepStmts=true&useServerPrepStmts=true&rewriteBatchedStatements=true&ssl=true&sslfactory=org.postgresql.ssl.NonValidatingFactory&sslmode=require
```

Comment:
It might be worth keeping both commented out here as one is ssl and one is not and if anyone is running a local pg they might not have ssl setup.